modules/olm-reinstall.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+:_content-type: PROCEDURE
+[id="olm-reinstall_{context}"]
+= Reinstalling Operators after failed uninstallation
+
+You must successfully and completely uninstall an Operator prior to attempting to reinstall the same Operator. Failure to fully uninstall the Operator properly can leave resources, such as a project or namespace, stuck in a "Terminating" state and cause "error resolving resource" messages. For example:
+
+.Example `Project` resource description
+----
+...
+    message: 'Failed to delete all resource types, 1 remaining: Internal error occurred:
+      error resolving resource'
+...
+----
+
+These types of issues can prevent an Operator from being reinstalled successfully.
+
+[WARNING]
+====
+Forced deletion of a namespace can lead to unstable or unpredictable cluster behavior, so it is better to try to find related resources that might be preventing the namespace from being deleted. For more information, see the link:https://access.redhat.com/solutions/4165791[Red Hat Knowledgebase Solution #4165791].
+====
+
+The following procedure shows how to troubleshoot when an Operator cannot be reinstalled because an existing custom resource definition (CRD) from a previous installation of the Operator is preventing a related namespace from deleting successfully.
+
+.Procedure
+
+. Check if there are any namespaces related to the Operator that are stuck in "Terminating" state:
++
+[source,terminal]
+----
+$ oc get namespaces
+----
++
+.Example output
+----
+operator-ns-1                                       Terminating
+----
+
+. Check if there are any CRDs related to the Operator that are still present after the failed uninstallation:
++
+[source,terminal]
+----
+$ oc get crds
+----
+
+. If there are any CRDs that should have been deleted after uninstalling an Operator, delete the CRD:
```

Comment:
There is a strong possibility that simply deleting the CRD definition will not help get things into a clean state.  In my experience this is usually a multi-step debut process and I am not sure how deep into it we want to get in the docs.

Reply:
Yes, they should also delete the related CR(the actual instances), but it's hard to describe the CR. Maybe we can describe an example here.